packages/account/src/Styles/account.scss

```diff
@@ -687,8 +693,9 @@ $MIN_HEIGHT_FLOATING: calc(
                     margin-top: 3.2rem;
 
                     @include mobile-or-tablet-screen {
-                        margin: 1.6rem 0;
+                        width: 568px;
```

Comment:
Hi @Dhruv-deriv 👋 
Instead of `px` can you please use `rem`

Thank you

Reply:
hey @fasihali-deriv i just noticed its not needed by my POI section can i remove it ?